lib/spdy-transport/protocol/http2/constants.js

```diff
@@ -46,7 +47,7 @@ exports.flags = {
   PRIORITY: 0x20
 };
 
-exports.settings = {
+exports.settings_server = {
```

Comment:
Are the `settings` values different for client and server? What's the point in renaming it?